src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13172,3 +13172,53 @@ def test_aks_disable_ssh(self, resource_group, resource_group_location):
 
         # delete
         self.cmd('aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest',
+                                    location='westus2', preserve_default_location=True)
+    def test_aks_check_network(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=88488831&view=results) to validate the change.

Reply:
I already triggered live test [here](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=88439744&view=results) and I believe it did pass. The recording in this PR was download from the artifacts. Let me know if I'm missing something?